src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -50,8 +50,8 @@
     public static final String VALID_REPEATABLE_DATE = " " + PREFIX_REPEATABLE_DATE + "01-01-2020";
     public static final String VALID_REPEATABLE_INTERVAL = " " + PREFIX_REPEATABLE_INTERVAL + "WEEKLY";
     public static final String INVALID_DESCRIPTION = " " + PREFIX_DESCRIPTION + "";
-    public static final String INVALID_DEADLINE_DATE = " " + PREFIX_DEADLINE_DATE + "01012020";
-    public static final String INVALID_REPEATABLE_DATE = " " + PREFIX_REPEATABLE_DATE + "01012020";
+    public static final String INVALID_DEADLINE_DATE = " " + PREFIX_DEADLINE_DATE + "01/01-2020";
+    public static final String INVALID_REPEATABLE_DATE = " " + PREFIX_REPEATABLE_DATE + "01-01/2020";
```

Comment:
Maybe this line can change INVALID_REPEATABLE_DATE -> INVALID_EVENT_DATE.

can't merge with deadline one because of the different prefix.

Reply:
Good spot haha, will update the other occurrences too.

Reply:
@vevek I went ahead and changed occurrences of REPEATABLE to EVENT, I think this makes it clearer.